spec/grape_entity/entity_spec.rb

```diff
@@ -29,9 +29,11 @@
         end
 
         it 'makes sure that :format_with as a proc cannot be used with a block' do
-          # rubocop:disable Style/BlockDelimiters
-          expect { subject.expose :name, format_with: proc {} do p 'hi' end }.to raise_error ArgumentError
-          # rubocop:enable Style/BlockDelimiters
+          expect do
```

Comment:
I cleaned this one up, but I wasn't sure about the other development dependency ones.